public/app/features/admin/ServerStatsCard.tsx

```diff
@@ -3,26 +3,56 @@ import React from 'react';
 import Skeleton from 'react-loading-skeleton';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { CardContainer, Stack, useStyles2 } from '@grafana/ui';
+import { Card, Stack, useStyles2, Tooltip, Icon } from '@grafana/ui';
 
 export interface Props {
-  content: Array<Record<string, string | number | undefined>>;
+  content: Array<Record<string, string | number | boolean | undefined>>;
   isLoading?: boolean;
   footer?: JSX.Element | boolean;
 }
 
 export const ServerStatsCard = ({ content, footer, isLoading }: Props) => {
   const styles = useStyles2(getStyles);
   return (
-    <CardContainer className={styles.container} disableHover>
+    <Card className={styles.container}>
       {content.map((item, index) => (
-        <Stack key={index} justifyContent="space-between" alignItems="center">
-          <span>{item.name}</span>
-          {isLoading ? <Skeleton width={60} /> : <span>{item.value}</span>}
-        </Stack>
+        <>
+          {item.tabbed ? (
```

Comment:
we don't need a huge ternary here just to add the ` - `, especially when the rest of the content is exactly the same

just do `{item.tabbed && ' - '}` inline

tbh i don't even think you need the ` - `, you could just give the span some left margin if it's indented.

```tsx
<span className={cx({
  [styles.indent]: item.tabbed
)}>{item.name}</span>

...
const getStyles = (theme: GrafanaTheme2) => {
  return {
    indent: css({
      marginLeft: theme.spacing(1),
    })
  }
}
```

Reply:
@ashharrison90 so this would 
```
<span className={cx({
  [styles.indent]: item.tabbed
)}>{item.name}</span>
```

only be applied if the `item.tabbed` is set

Reply:
yep exactly 👍

Reply:
Cool! fixed so that the indent is there now without the `-` as well